src/main/java/uk/ac/ebi/embl/flatfile/writer/embl/FTWriter.java

```diff
@@ -30,33 +30,59 @@
 /** Flat file writer for the feature table.
  */
 public class FTWriter extends FlatFileWriter {
-	
-	public FTWriter(Entry entry, boolean sortFeatures, 
-			boolean sortQualifiers, WrapType wrapType) {
+
+	public FTWriter(Entry entry, boolean sortFeatures,
+					boolean sortQualifiers, WrapType wrapType) {
 		super(entry, wrapType);
 		this.sortFeatures = sortFeatures;
 		this.sortQualifiers = sortQualifiers;
 	}
 
+	public FTWriter(Entry entry, boolean sortFeatures,
+					boolean sortQualifiers, boolean excludeSource, WrapType wrapType) {
+		super(entry, wrapType);
+		this.sortFeatures = sortFeatures;
+		this.sortQualifiers = sortQualifiers;
+		this.excludeSource = excludeSource;
+	}
+
 	private boolean sortFeatures;
 	private boolean sortQualifiers;
+	private boolean excludeSource;
 
 	public boolean write(Writer writer) throws IOException {
 		Vector<Feature> features = new Vector<Feature>(entry.getFeatures());
 		if (features == null ||
 			features.size() == 0) {
 			return false;
 		}
+		if (excludeSource && allSourceFeatures(features)) {
+			return false;
+		}
 		if (sortFeatures) {
 			Collections.sort(features);
 		}
 		new FHWriter(entry).write(writer);
 		for (Feature feature : features) {
+			if (excludeSource && isSourceFeature(feature))
+				continue;
             writeFeature(writer, feature);
         }
 		return true;		
 	}
 
+	private boolean isSourceFeature(Feature feature) {
+		return feature.getName().equals(Feature.SOURCE_FEATURE_NAME);
+	}
+
+	private boolean allSourceFeatures(Vector<Feature> features) {
+		for (Feature feature : features) {
+			if (!isSourceFeature(feature))
+				return false;
```

Comment:
- Just a code formatting comment. For new code we should follow Google Java code style completely and for old code as much as we can. E.g. we should use braces even when they are optional (https://google.github.io/styleguide/javaguide.html#s4.1.1-braces-always-used). Not having braces may result in 'hard to see' bugs.